charts/openfaas-cloud/values.yaml

```diff
@@ -0,0 +1,95 @@
+# Default values for openfaas-cloud.
+# This is a YAML-formatted file.
+
+## ofBuilder is the section for of-builder configuration specific settings
+ofBuilder:
+
+  ## names is the name of the deployment for the of-builder component
+  name: of-builder
+
+  ## image is the container image that is used for the deployment of of-builder
+  image: openfaas/of-builder
```

Comment:
We have this in one field with faas-netes to help with use with CD tools like Flux and Argo - https://github.com/openfaas/faas-netes/blob/master/chart/openfaas/values.yaml#L24

It's easier for them to work with.